app/components/ui/NewsArticles/NewsArticles.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState, useEffect } from "react";
+import {
+  CarouselProvider,
+  Slider,
+  Slide,
+  ButtonBack,
+  ButtonNext,
+  DotGroup,
+} from "pure-react-carousel";
+
+import * as dataService from "../../../utils/DataService";
+
+import styles from "./NewsArticles.module.scss";
+
+interface NewsArticle {
+  id: string;
+  headline: string;
+  effective_date: string;
+  expiration_date: string;
+  body: string;
+  priority: number;
+  url?: string;
+}
+
+const formatDate = (date: string): string =>
+  new Date(date).toLocaleDateString("en-US", {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+  });
+
+const ArticleItem = ({ article }: { article: NewsArticle }) => (
+  <div className={styles.articleItem}>
+    <div className={styles.newsContent}>
+      <p className={styles.articleTitle}>
+        {article.headline} <span className={styles.dateSeperator}>|</span>{" "}
+        <span className={styles.articleDate}>
+          {formatDate(article.effective_date)}
+        </span>
+      </p>
+      <p className={styles.articleBody}>{article.body}</p>
+    </div>
+    {article.url && (
+      <a href={article.url} className={styles.articleLink}>
+        <span className={styles.linkText}>Read more →</span>
+        <span className={styles.linkText_mobile}>More</span>
+      </a>
+    )}
+  </div>
+);
+
+export const NewsArticles = () => {
+  const [collapseCarousel, setCollapseCarousel] = useState(false);
+  const [breakingNewsArticles, setBreakingNewsArticles] = useState<
+    NewsArticle[]
+  >([]);
+
+  useEffect(() => {
+    const fetchBreakingNewsArticles = async () => {
+      const response = await dataService.get("/api/news_articles");
+      const { news_articles }: { news_articles: NewsArticle[] } = response;
+      setBreakingNewsArticles(news_articles);
+    };
+
+    fetchBreakingNewsArticles();
+  }, []);
+
+  if (breakingNewsArticles.length === 0) return null;
+
+  return (
+    <div
+      className={`${styles.container} ${
+        collapseCarousel && styles.carouselCollapsed
+      }`}
+    >
+      <div>
+        <div className={styles.articlesHeader}>
+          <p className={styles.title}>News</p>
+          <button
+            type="button"
+            className={`${styles.toggleCarouselButton}`}
+            onClick={() => setCollapseCarousel(!collapseCarousel)}
+          >
+            <i
+              className={`material-icons ${
+                collapseCarousel
+                  ? styles.expandCarouselChev
+                  : styles.collapseCarouselChev
+              }`}
+            >
+              keyboard_arrow_down
+            </i>
+          </button>
+        </div>
+        {!collapseCarousel && (
```

Comment:
If you do the collapsing purely in CSS and not JS, then we can put in a nice animation for the collapsing that wouldn't be possible if you actually removed the DOM element. Here's an article that shows one approach of doing a CSS animation for a collapsible component: https://jdsteinbach.com/css/holy-grail-css-animation/